.github/workflows/android_e2e.yml

```
@@ -0,0 +1,154 @@
+            if $ANDROID_SDK_ROOT/platform-tools/adb shell getprop sys.boot_completed | grep 1 > /dev/null; then
+              echo "✅ Emulator booted"
+              break
+            fi
+            echo "Waiting for boot... ($i/30)"
+            sleep 5
+          done
+      
+          echo "🌐 Starting Metro..."
+          pnpm start --reset-cache &
+          METRO_PID=$!
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)